### DISCUSSION TITLE
removal of ext-workspace-unstable-v1

### DISCUSSION THREAD

**Discussion by fuyarovh** [hyprwm/Hyprland#3215] [Q&A]
In bb09334 ext-workspace-unstable-v1 was removed, partly because [the associated wlroots merge request](https://gitlab.freedesktop.org/wlroots/wlr-protocols/-/merge_requests/35) was closed. Hovewer there is another [merge request for wayland](https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/40) for the same protocol that is still open. Was this considered in the decision to remove the protocol from hyprland?

**Comment by vaxerski** [comment]
yes.
